packages/worker/src/core.ts

```diff
@@ -15,9 +15,9 @@ import { compileServerOptions, getDefaultServerOptions, RequiredServerOptions, S
 import { DefaultLogger, LogEntry, Logger, LogTimestamp, timeOfDayToBigint } from './logger';
 import * as errors from './errors';
 import { temporal } from '@temporalio/proto';
-import History = temporal.api.history.v1.History;
 import { byteArrayToBuffer } from './utils';
 import { MakeOptional } from '@temporalio/common/src/type-helpers';
+import History = temporal.api.history.v1.History;
```

Comment:
Let's export this as the interface instead:
```ts
export type History = temporal.api.history.v1.IHistory;
```

Reply:
I don't think so - it should be the actual reified type, which ensures that it's actually been parsed properly and doesn't have undefined fields that can't actually be that way in a real history